frontend/packages/operator-lifecycle-manager/src/components/modals/uninstall-operator-modal.tsx

```diff
@@ -121,39 +226,341 @@ export const UninstallOperatorModal: React.FC<UninstallOperatorModalProps> = ({
+          name="delete-all-operands"
+          label={t('olm~Delete all operand instances for this operator')}
+          checked={isDeleteAllOperandsSelected}
+        />
+      </span>
+    )
+  );
+};
+
+const OperandsLoadAlert: React.FC<{ operandsLoadedErrorMessage: string }> = ({
+  operandsLoadedErrorMessage,
+}) => {
+  const { t } = useTranslation();
+  return (
+    <Alert variant="warning" className="co-alert" title={t('olm~Cannot load Operands')} isInline>
+      <p>
+        {t(
+          'olm~There was an error loading operands for this operator. Operands will need to be deleted manually.',
+        )}
+        <br />
+        {operandsLoadedErrorMessage}
+      </p>
+    </Alert>
+  );
+};
+
+const OperatorUninstallSuccess: React.FC<{ name: string; namespace: string }> = ({
+  name,
+  namespace,
+}) => {
+  const { t } = useTranslation();
+  return (
+    <Alert
+      variant="success"
+      className="co-alert"
+      title={t('olm~Successfully Uninstalled Operator')}
+      isInline
+    >
+      <p>
+        <Trans t={t} ns="olm">
+          Operator <strong>{{ name }}</strong> successfully uninstalled from{' '}
+          <strong>{{ namespace }}</strong>.
+        </Trans>
+      </p>
+    </Alert>
+  );
+};
+
+const OperatorUninstallError: React.FC<{ errorMessage: string }> = ({ errorMessage }) => {
+  const { t } = useTranslation();
+  return (
+    <Alert
+      variant="danger"
+      className="co-alert"
+      title={t('olm~Error Uninstalling Operator')}
+      isInline
+    >
+      <p>
+        {t('olm~There was an error uninstalling the operator.')}
+        <br />
+        {errorMessage}
+      </p>
+    </Alert>
+  );
+};
+
+const OperandDeletionErrors: React.FC<{
+  operandDeletionErrors: OperandWithErrorProps[];
+  csvName: string;
+  cancel?: () => void;
+}> = ({ operandDeletionErrors, csvName, cancel }) => {
+  const { t } = useTranslation();
+  return (
+    <Alert variant="danger" className="co-alert" title={t('olm~Error Deleting Operands')} isInline>
+      <p>
+        <Trans t={t} ns="olm">
+          There were errors deleting the following Operands, they will need to be deleted manually:
+        </Trans>
+      </p>
+      <OperandErrors
+        operandsWithErrors={operandDeletionErrors}
+        csvName={csvName}
+        cancel={cancel} // for breadcrumbs & cancel modal when clicking on operand links
+      />
+    </Alert>
+  );
+};
+
+const OperandDeletionSuccess: React.FC<{ name: string; namespace: string }> = ({
+  name,
+  namespace,
+}) => {
+  const { t } = useTranslation();
+  return (
+    <Alert
+      variant="success"
+      className="co-alert"
+      title={t('olm~Successfully deleted all Operand Instances')}
+      isInline
+    >
+      <p>
+        <Trans t={t} ns="olm">
+          All Operand instances for Operator <strong>{{ name }}</strong> in{' '}
+          <strong>{{ namespace }}</strong> have been deleted.
```

Comment:
Not sure if this msg makes sense for operands without namespace?

Reply:
in the `OperandsTable` we are checking for `operand.metadata.namespace`

Reply:
`namespace` will be either 'all-namespaces' or `subscription.metadata.namespace`, pls see [here](https://github.com/openshift/console/blob/97f0c26dbfbf67310b083b41bc717ed672018fee/frontend/packages/operator-lifecycle-manager/src/components/modals/uninstall-operator-modal.tsx#L218).